processor/src/main/java/com/linecorp/decaton/processor/runtime/internal/PartitionContext.java

```diff
@@ -55,6 +55,7 @@ public class PartitionContext implements AutoCloseable {
     @Getter
     @Setter
     private boolean revoking;
+    private long latestConsumedOffset;
```

Comment:
This and `lastCommittedOffset` needs to be `volatile` because they can be written and read by different threads at arbitrary timing (decaton's thread and metric exporting server's thread)